frontend/src/app/modules/bim/bcf/api/viewpoints/bcf-viewpoint-item.interface.ts

```diff
@@ -0,0 +1,10 @@
+import {BcfViewpointInterface} from "core-app/modules/bim/bcf/api/viewpoints/bcf-viewpoint.interface";
+
+export interface BcfViewpointItem {
+  /** The URL of the viewpoint, if persisted */
+  href?:string|null;
+  /** URL (persisted or data) to the snapshot */
+  snapshotURL:string;
+  /** The loaded snapshot, if exists */
```

Comment:
In the comment you mean "viewpoint" and not "snapshot". The snapshot is the pixel graphic of a viewpoint. The viewpoint is some data (JSON) plus the viewpoint (the pixel graphic)